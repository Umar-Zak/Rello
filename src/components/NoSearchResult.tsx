import React from "react";
import "styled-components"
import styled from "styled-components/native";
function NoSearchResult({text}: {text?: string}) {
    return (
       <Container>
        <Text>
           {text? text : "No results"}
        </Text>
       </Container>
    );
}

export default NoSearchResult;

const Container = styled.View`
    flex: 1;
    align-items: center;
    justify-content: center;
`

const Text = styled.Text`
    
`