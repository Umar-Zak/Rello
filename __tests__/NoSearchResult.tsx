import * as React from 'react';
import renderer from "react-test-renderer"
import NoSearchResult from '../src/components/NoSearchResult';
import "jest-styled-components"

describe("", () => {
    it("has 1 child", () => {
       const tree = renderer.create(<NoSearchResult/>).toJSON()
       expect(tree).toMatchSnapshot()
       //@ts-ignore
       expect(tree.children.length).toBe(1)
    })
})