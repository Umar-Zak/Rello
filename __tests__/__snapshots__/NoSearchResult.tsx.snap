// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has 1 child 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {},
      ]
    }
  >
    No results
  </Text>
</View>
`;
